cla_backend/templates/admin/base_site.html

```diff
@@ -2,12 +2,18 @@
 {% load i18n admin_static %}
 
 {% block extrahead %}{{ block.super }}
+    <link rel="shortcut icon" href="{% static 'images/gov.uk_logotype_crown.ico' %}" type="image/x-icon">
+    <link rel="mask-icon" href="{% static 'images/gov.uk-mask-icon.svg' %}" color="#0b0c0c">
+    <link rel="apple-touch-icon" sizes="180x180" href="{% static 'images/gov.uk-apple-touch-icon-180x180.png' %}">
+    <link rel="apple-touch-icon" sizes="167x167" href="{% static 'images/gov.uk-apple-touch-icon-167x167.png' %}">
+    <link rel="apple-touch-icon" sizes="152x152" href="{% static 'images/gov.uk-apple-touch-icon-152x152.png' %}">
+    <link rel="apple-touch-icon" href="{% static 'images/govuk-apple-touch-icon.png' %}">
```

Comment:
Please double check if exists as I couldn't find it

Reply:
Thanks Jav, I have resolved this typo! 😄